modules/social_features/social_featured_content/social_featured_content.install

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the social_featured_content module.
+ */
+
+/**
+ * Add translations support for "Featured Content" custom block fields.
+ */
+function social_featured_content_update_10200() {
```

Comment:
Since we add this to 10.3 milestone, need to rename the number to start with 103

Reply:
fixed